content/tasks/third/index.md

```diff
@@ -0,0 +1,74 @@
+# Santacloud reparieren
```

Comment:
This also uses the wrong heading level

Reply:
The description of Task 1 and Task 2 uses a headline level 1, as does my section. Could you clarify in more detail what specifically is wrong in this context?